.travis.yml

```diff
@@ -1,12 +1,13 @@
 language: python
-python:
-  - 2.7
-  - 3.3
-  - 3.4
+env:
+  - TOXENV=py26
```

Comment:
I like being able to view a matrix of results in Travis. There are times when a bug will only manifest itself in certain Python versions, and making that information easily discoverable is immensely valuable. That's why I'm OK with duplicating this information. If there is some method by which we can de-duplicate this information and still get a matrix of easily discoverable results, then that would be even cooler. But I don't know how to do that. I just don't know enough about the interactions between Travis and tox.